src/main/java/com/endava/soa_5/tools/PropertyReader.java

```diff
@@ -0,0 +1,43 @@
+package com.endava.soa_5.tools;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+
+import static com.endava.soa_5.tools.Constants.CONFIGURATION_FOLDER;
+
+public class PropertyReader {
+    private Properties properties;
+    private InputStream inputStream;
+    private String propertyValue;
+
+    public PropertyReader(String fileName) {
+        properties = new Properties();
+        properties = loadConfiguration(fileName);
+    }
+
+    public Properties loadConfiguration(String fileName) {
+        inputStream = getClass().getClassLoader().getResourceAsStream(CONFIGURATION_FOLDER + fileName);
+        try {
+            properties.load(inputStream);
+        } catch (IOException ioException) {
+            ioException.printStackTrace();
+        }
+        return properties;
+    }
+
+    public String getPropertyValue(String key) {
+        try {
+            String controlValue = properties.getProperty(key);
+            if (controlValue != null) {
+                propertyValue = controlValue;
```

Comment:
please consider that instead creating new variable, can be simply returned controlValue in if(), else can be removed, and in the end returned null (so in case that something was happening it will return null)

Reply:
Done!